src/types/subscription.ts

```diff
@@ -89,6 +89,10 @@ export enum BalanceChangeRole {
   Receiver = 'Receiver',
   /** Account was having an asset amount closed to it */
   CloseTo = 'CloseTo',
+  /** Account was creating an asset and holds the full asset supply */
+  AssetCreator = 'AssetCreator',
+  /** Account was destroying an asset and has removed the full asset supply from circulation */
```

Comment:
Might be worth mentioning that the balance change amount will be 0, but there is a side-effect that the creator loses all of the supply.

Reply:
I wasn't exactly sure the best place to put this. I did mention that in the readme, however I'll also add here.